### DISCUSSION TITLE
Lesson 1: Environment bug

### DISCUSSION THREAD

**Discussion by bozinovicn** [smartcontractkit/full-blockchain-solidity-course-py#1481] [Q&A]
Hello,
I'm having a problem with environment section. Whenever I try to change from JavaScript VM to Injected Web3 it just stays JavaScript. Does anyone has an idea what's going on?

**Comment by cromewar** [comment]
Hello @bozinovicn 
This could be a error with the cache of you browser, so please try the following.

1. Delete all the cache, cookies and navigation data from you browser.
2. Try a browser not based in Chromium like Firefox.


I hope this info might help.
